src/piaf/models.py

```diff
@@ -1,27 +1,44 @@
 from django.db import models
 from django.contrib.auth.models import User
 
+STATUSES = [('pending', 'pending'), ('progress', 'progress'), ('completed', 'completed')]
+
 
 class Article(models.Model):
-    CHOICES = [('pending', 'pending'), ('progress', 'progress'), ('completed', 'completed')]
     name = models.CharField(max_length=100)
-    status = models.CharField(max_length=10, choices=CHOICES, default='pending')
+
+
+class ParagraphBatch(models.Model):
     user = models.ForeignKey(User, null=True, on_delete='null')
     participated_at = models.DateField(auto_now=True)
+    status = models.CharField(max_length=10, choices=STATUSES,
+                              default='pending')
```

Comment:
Should we link a batch to an article ? 
Like using "models.ForeignKey(Article, null=True, on_delete='null')"

Reply:
Okay. it is linked through paragraph. Okay, fine for me